CustomerData/CustomerBehaviour.php

```diff
@@ -13,9 +13,13 @@ class CustomerBehaviour implements SectionSourceInterface
 {
     private CurrentCustomer $currentCustomer;
 
+    private Session $customerBehaviourSession;
```

Comment:
How are CurrentCustomer and Session in injected here?
I'm less familiar with this the Private Content concept, so trying to learn as well :)